docs/setup/install-with-keadm.md

```diff
@@ -2,51 +2,54 @@
 title: Installing KubeEdge with Keadm
 sidebar_position: 3
 ---
-Keadm is used to install the cloud and edge components of KubeEdge. It is not responsible for installing K8s and runtime.
 
-Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kubernetes-compatibility) to get **Kubernetes compatibility** and determine what version of Kubernetes would be installed.
+Keadm is used to install the cloud and edge components of KubeEdge. It does not handle the installation of Kubernetes and its runtime environment.
 
-## Limitation
+Please refer to [Kubernetes compatibility](https://github.com/kubeedge/kubeedge#kubernetes-compatibility) documentation to check **Kubernetes compatibility** and ascertain the Kubernetes version to be installed.
 
-- Need super user rights (or root rights) to run.
+## Limitation
 
+- It Requires super user rights (or root rights) to run.
 
 ## Install keadm
 
-There're three ways to download a `keadm` binary
+There're three ways to download the `keadm` binary:
```

Comment:
The component name here is `Keadm`, but `keadm` is used when executing such as `keadm join` on the command line and installing packages